pennylane_lightning/lightning_kokkos/_state_vector.py

```diff
@@ -201,9 +201,14 @@ def _apply_lightning_controlled(self, operation):
             inv = False
             param = operation.parameters
             method(control_wires, control_values, target_wires, inv, param)
-        else:
-            raise qml.DeviceError(
-                "No gate operation supplied and controlled matrix not yet supported"
+        else:  # apply gate as an n-controlled matrix
+            method = getattr(state, "applyControlledMatrix")
+            method(
+                qml.matrix(operation.base),
+                control_wires,
+                control_values,
+                target_wires,
+                False,
```

Comment:
Could you also add a code comment about this `inv == False` for the future reference? (this is a missing feature that we'll tackle later)
```suggestion
                inv,
```

Reply:
updated